packages/myst-cli/src/transforms/doi.spec.ts

```diff
@@ -1,26 +1,44 @@
 import { describe, expect, it } from 'vitest';
 import { Session } from '../session';
-import { getDoiOrgBibtex } from './dois';
+import { resolveDOIAsBibTeX, resolveDOIAsCSLJSON } from './dois';
 
-const PRIESTLEY_1972 =
-  '@article{PRIESTLEY_1972, title={On the Assessment of Surface Heat Flux and Evaporation Using Large-Scale Parameters}, volume={100}, ISSN={1520-0493}, url={http://dx.doi.org/10.1175/1520-0493(1972)100<0081:OTAOSH>2.3.CO;2}, DOI={10.1175/1520-0493(1972)100<0081:otaosh>2.3.co;2}, number={2}, journal={Monthly Weather Review}, publisher={American Meteorological Society}, author={PRIESTLEY, C. H. B. and TAYLOR, R. J.}, year={1972}, month=feb, pages={81–92} }';
-describe('DOI Resolvers', () => {
+const PRIESTLEY_1972_CSL_JSON = [
+  {
+    'container-title': 'Monthly Weather Review',
+    author: [
+      { given: 'C. H. B.', family: 'PRIESTLEY' },
+      { given: 'R. J.', family: 'TAYLOR' },
+    ],
+    DOI: '10.1175/1520-0493(1972)100<0081:otaosh>2.3.co;2',
+    type: 'article-journal',
+    issue: '2',
+    issued: { 'date-parts': [[1972, 2]] },
+    page: '81-92',
+    publisher: 'American Meteorological Society',
+    title: 'On the Assessment of Surface Heat Flux and Evaporation Using Large-Scale Parameters',
+    volume: '100',
+  },
+];
```

Comment:
I simplified this to the subset of fields common to both resolvers. I think that's OK.